easybuild/easyconfigs/t/TensorRT/TensorRT-4.0.1.6-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+name = 'TensorRT'
+version = '4.0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://developer.nvidia.com/tensorrt'
+description = """NVIDIA TensorRT is a platform for high-performance deep learning inference"""
+
+sources = ['%(name)s-%(version)s.Ubuntu-16.04.4.x86_64-gnu.cuda-9.2.cudnn7.1.tar.gz']
```

Comment:
@akesandgren Maybe mention that downloading manually via https://developer.nvidia.com/nvidia-tensorrt-4x-download is required in a comment?